buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersAware.java

```diff
@@ -18,9 +18,13 @@ default void useCluster(ElasticsearchCluster cluster) {
             );
         }
 
-        cluster.getNodes().stream().flatMap(node -> node.getDistributions().stream()).forEach( distro ->
+        cluster.getNodes().stream().flatMap(node -> node.getDistributions().stream()).forEach(distro ->
             dependsOn(distro.getExtracted())
         );
         getClusters().add(cluster);
     }
+
+    default void beforeStart() {
```

Comment:
Perhaps I'm missing something but I can't find anywhere where this is actually called.

Reply:
Good catch, it wasn't and I didn't test all use-cases so didn't notice it. It's fixed now.